bin/deploy

```diff
@@ -10,14 +10,16 @@ source "${SCRIPT_DIR}/../etc/settings.cfg.sh"
 EUPS_VERSION=${EUPS_VERSION:-2.1.4}         # Version of EUPS to install
 EUPS_GITREV=${EUPS_GITREV:-""}
 EUPS_GITREPO=${EUPS_GITREPO:-https://github.com/RobertLuptonTheGood/eups.git}
-MINICONDA2_VERSION=${MINICONDA2_VERSION:-4.3.21} # Version of Miniconda to install
-MINICONDA3_VERSION=${MINICONDA3_VERSION:-4.3.21} # Version of Miniconda to install
+LSST_PYTHON_VERSION=${LSST_PYTHON_VERSION:-3}
+MINICONDA_VERSION=${MINICONDA_VERSION:-4.3.21} # Version of Miniconda to install
```

Comment:
This version is only relevant for building up the url to download an installer and for establishing an pseudo-ABI for eups "tarball" products.  `conda` will install and configure the correct python version to match an environment export regardless of what version of the installer package was used.  This can be demonstrated by installing the `7c8e67` env files with 4.3.21 and checking the python interpreter version -- it will transparently switch back to py 3.5.